jsplot/src/main/java/tech/tablesaw/plotly/components/Marker.java

```diff
@@ -155,6 +163,39 @@ public String asJavascript() {
     return context;
   }
 
+  static class Color {
+
+    final int r, g, b;
```

Comment:
What about adding the colorScaleCustomRange value to the Color class? It looks like colors and values are always added at the same time.

That will allow you to only add one field to the MarkerBuilder (not two).